docs/Modders/Mod-file-structure/3-Map-related-JSON-files.md

```diff
@@ -41,11 +41,11 @@ Each improvement has the following structure:
 | terrainsCanBeFoundOn | List | empty | [Terrains](#terrainsjson) that allow this resource |
 | techRequired | String | none | The name of the technology required to build this improvement |
 | uniqueTo | String | none | The name of the nation this improvement is unique for |
-| [`<stats>`](#stats) | Integer | Default | Per-turn bonus yield for the tile |
-| turnsToBuild | Integer | Default | Number of turns a worker spends building this (leaving it empty makes improvement unbuildable) |
+| [`<stats>`](#stats) | Integer | 0 | Per-turn bonus yield for the tile |
+| turnsToBuild | Integer | -1 | Number of turns a worker spends building this (note: if -1, the improvement is unbuildable) |
```

Comment:
Should probably state that leaving it empty or -1 makes it unbuildable. I know it's a weird thing to point out (leaving it blank leaves it at the default of -1), but specifying that explicitly for mechanical effects like this may be helpful to mods who might not realize. ~~Might also be helpful to specify that 0 always makes it build in 1 turn, but Idk if that's too much information~~

Reply:
Also, I commented on the wrong PR. Goddammit. Looks like you already mentioned the 0 part

Reply:
Actually, you took care of that as a note. Nevermind